guide/02-api-overview/overview24.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Both <code>wm.add_table()</code> and <code>wm.move_to_basemap()]</code> in the methods table has extra brackets; And the <code>m.basemap.move_from_basemap()</code> could have layer_index as parameter, similar as move_to_basemap() function :)</p><p> </p><p>For the <code>wm.basemap.basemap_gallery</code> in the property table - should we remove 3rd row as there is a same basemap_gallery row below?</p><p> </p><p>Also missing the return type for <code>offline_areas</code> in property table - OfflineMapAreaManager class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<ul><li>first comment addressed by Noah's pr</li><li>I think we should keep them because the property exists on both the WebMap and MapView classes</li><li>added link to the OfflineMapAreaManager</li></ul><p> </p>